inappmessaging/USERGUIDE.md

```diff
@@ -112,7 +112,7 @@ Message are displayed if the triggers/conditions for the campaign are satisfied
 ![Display message](images/what-is-inapp.png)
 
 ## <a name="integration"></a> SDK Integration
-### #1 Include Artifactory in your project, this should be added in your project root `build.gradle` file. Note: Crimson House Intra connection is needed.
+### #1 Include Artifactory in your project, this should be added in your project root `build.gradle` file.
```

Comment:
Should probably also update this description: `Include JCenter repo in your project`

Reply:
updated description